copy/multiple.go

```diff
@@ -32,6 +32,9 @@ type instanceCopy struct {
 	op           instanceCopyKind
 	sourceDigest digest.Digest
 
+	// Fields which we want to preserve
+	artifactType string
```

Comment:
(2) it is a bit awkward for `instanceCopy` to be a carrier of data the copy process does not inherently care about.

I’d prefer this to only be a `cloneArtifactType`, in the section below, and for the preservation of the field on “ordinary copies” to be implicit. (Even the `clonePlatform`/`cloneAnnotations` are not really _relevant_ to this copy code, but we have them ~so that we only call `.Instance()` at one place.)

Reply:
Okay, changing it.